src/modules/rendertodolist.js

```diff
@@ -0,0 +1,13 @@
+const todoUl = document.querySelector('.todo-list');
+const renderTodoList = ({ id, description, ischeck }) => {
+  const liItems = document.createElement('li');
+  liItems.innerHTML = `
+    <input id="${id}" type="checkbox">
+    <span class="editTodo" id="${id}">${description}</span>
+    <button class="delete-todo" id="${id}" type="button">Delete</button>
+  `;
+
+  todoUl.append(liItems);
```

Comment:
- [ ] Kindly this should have an action to make a book completed if the user select the checkbox. add that and let see your project approve :handshake:

Reply:
Hi @vic778 
The project submission is for day 3.
Kindly check the day 3 project requirement for your review and approval. 
Thank you!